yarn-project/ivc-integration/src/rollup_ivc_integration.test.ts

```diff
@@ -58,7 +59,7 @@ describe('Rollup IVC Integration', () => {
 
     const [bytecodes, witnessStack, tailPublicInputs] = await generate3FunctionTestingIVCStack();
     clientIVCPublicInputs = tailPublicInputs;
-    const proof = await proveClientIVC(bbBinaryPath, clientIVCWorkingDirectory, witnessStack, bytecodes, logger);
+    const proof = await proveClientIVCWASM(bytecodes, witnessStack, 16);
```

Comment:
you should be able to run `LOG_LEVEL=debug yarn test` to test this rigged thing given the modifications in package.json